kubectl-plugin/pkg/util/generation/generation_test.go

```diff
@@ -64,24 +65,92 @@ func TestGenerateRayClusterApplyConfig(t *testing.T) {
 
 	result := testRayClusterSpecObject.GenerateRayClusterApplyConfig()
 
-	assert.Equal(t, testRayClusterSpecObject.Name, result.Name)
-	assert.Equal(t, testRayClusterSpecObject.Namespace, result.Namespace)
-	assert.Equal(t, testRayClusterSpecObject.Labels, labels)
-	assert.Equal(t, testRayClusterSpecObject.Annotations, annotations)
-	assert.Equal(t, testRayClusterSpecObject.RayVersion, result.Spec.RayVersion)
-	assert.Equal(t, testRayClusterSpecObject.Image, result.Spec.HeadGroupSpec.Template.Spec.Containers[0].Image)
-	assert.Equal(t, resource.MustParse(*testRayClusterSpecObject.HeadCPU), *result.Spec.HeadGroupSpec.Template.Spec.Containers[0].Resources.Requests.Cpu())
-	assert.Equal(t, resource.MustParse(*testRayClusterSpecObject.HeadGPU), *result.Spec.HeadGroupSpec.Template.Spec.Containers[0].Resources.Requests.Name(corev1.ResourceName(util.ResourceNvidiaGPU), resource.DecimalSI))
-	assert.Equal(t, resource.MustParse(*testRayClusterSpecObject.HeadMemory), *result.Spec.HeadGroupSpec.Template.Spec.Containers[0].Resources.Requests.Memory())
-	assert.Equal(t, testRayClusterSpecObject.HeadRayStartParams, result.Spec.HeadGroupSpec.RayStartParams)
-	assert.Equal(t, "default-group", *result.Spec.WorkerGroupSpecs[0].GroupName)
-	assert.Equal(t, *testRayClusterSpecObject.WorkerGroups[0].WorkerReplicas, *result.Spec.WorkerGroupSpecs[0].Replicas)
-	assert.Equal(t, *testRayClusterSpecObject.WorkerGroups[0].NumOfHosts, *result.Spec.WorkerGroupSpecs[0].NumOfHosts)
-	assert.Equal(t, resource.MustParse(*testRayClusterSpecObject.WorkerGroups[0].WorkerCPU), *result.Spec.WorkerGroupSpecs[0].Template.Spec.Containers[0].Resources.Requests.Cpu())
-	assert.Equal(t, resource.MustParse(*testRayClusterSpecObject.WorkerGroups[0].WorkerGPU), *result.Spec.WorkerGroupSpecs[0].Template.Spec.Containers[0].Resources.Requests.Name(corev1.ResourceName(util.ResourceNvidiaGPU), resource.DecimalSI))
-	assert.Equal(t, resource.MustParse(*testRayClusterSpecObject.WorkerGroups[0].WorkerTPU), *result.Spec.WorkerGroupSpecs[0].Template.Spec.Containers[0].Resources.Requests.Name(corev1.ResourceName(util.ResourceGoogleTPU), resource.DecimalSI))
-	assert.Equal(t, resource.MustParse(*testRayClusterSpecObject.WorkerGroups[0].WorkerMemory), *result.Spec.WorkerGroupSpecs[0].Template.Spec.Containers[0].Resources.Requests.Memory())
-	assert.Equal(t, expectedWorkerRayStartParams, result.Spec.WorkerGroupSpecs[0].RayStartParams)
```

Comment:
Is `expectedWorkerRayStartParams` still used anywhere?

Reply:
good catch, removed. Too bad the linter isn't smarter :)